.github/PULL_REQUEST_TEMPLATE/pull_request_checklist.md

```diff
@@ -1,6 +1,6 @@
-## Describe your changes
+## Describe your changes(in 200 words)
```

Comment:
No need for this, I prefer long descriptions

Reply:
okk i dont have any knowlege in the domain

Reply:
Learn then.